extensions/stackdriver/metric/record.cc

```diff
@@ -25,6 +27,10 @@ namespace Metric {
 void record(bool is_outbound, const ::wasm::common::NodeInfo &local_node_info,
             const ::wasm::common::NodeInfo &peer_node_info,
             const ::Wasm::Common::RequestInfo &request_info) {
+
+  if (absl::StartsWith(request_info.user_agent, "kube-probe") || absl::StartsWith(request_info.user_agent, "Prometheus")) {
```

Comment:
@mandarjog @bianpengyuan where should we put this kinda logic in a non-hack branch?